Index: src/App.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import logo from './logo.svg';\r\nimport '@/App.css';\r\nimport { BrowserRouter as Router, Routes, Route } from 'react-router-dom'\r\n\r\nfunction App() {\r\n  return (\r\n    <></>\r\n  );\r\n}\r\n\r\nexport default App;\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/App.js b/src/App.js
--- a/src/App.js	(revision 0257f8c899432171c83e573a9fe62a20eae125dd)
+++ b/src/App.js	(date 1703740501387)
@@ -1,10 +1,10 @@
 import logo from './logo.svg';
-import '@/App.css';
+
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
 
 function App() {
   return (
-    <></>
+    <>hi</>
   );
 }
 
